InformationScripting/src/wrappers/NodeApi.cpp

```diff
@@ -28,15 +28,38 @@
 
 #include "../graph/PropertyMap.h"
 #include "../graph/InformationNode.h"
+#include "../graph/InformationEdge.h"
+#include "../graph/Graph.h"
 
 namespace InformationScripting {
 
 using namespace boost::python;
 
+// The following macro has unused local typdef so we disable the warning for this.
```

Comment:
Patrick will be very happy to see this :)

Is there some way to avoid the use of pragmas (without disabling the flag altogether)? Why does the macro have an unused typedef? Can't we hackaround that somehow?

Disabling warning just for this file, somehow in the .pro file, might be a better solution, if possible.